src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritConnection.java

```diff
@@ -94,6 +94,7 @@
     private AuthenticationUpdater authenticationUpdater = null;
     private final Set<ConnectionListener> listeners = new CopyOnWriteArraySet<ConnectionListener>();
     private int sshRxBufferSize = SSH_RX_BUFFER_SIZE;
+    private StringBuilder eventBuffer = null;
```

Comment:
@mawinter69 
Well, it's not @rinrinne says:

> Unfortunately it is not guaranteed that getLine method call is done only by the same thread during keeping this StringBuilder instance

I understand that it may be used by non-GerritConnection threads.